pages/linux/cpufreq-info.md

```diff
@@ -0,0 +1,35 @@
+# cpufreq-info
+
+> A tool to show CPU frequency information.
+
+- Show CPU frequency kernel information for all CPUs:
+
+`cpufreq-info`
+
+- Show CPU frequency kernel information for the specified CPU:
+
+`cpufreq-info -c {{cpu_number}}`
+
+- Show current CPU work frequency in a human-readable format, according to the kernel module:
```

Comment:
which kernel module ?

Reply:
The `cpufreq` kernel module. I've updated  the documentation.